frontend/src/components/create-user-page/CreateUser.tsx

```diff
@@ -1,3 +1,118 @@
-import React from "react";
+import React, { FormEvent, useContext, useState } from "react";
+import { LoginContext } from "../login/LoginManager";
 
-export const CreateUser: React.FunctionComponent = () => <></>;
+export const CreateUser: React.FunctionComponent = () => {
+  const loginContext = useContext(LoginContext);
+
+  const [formData, setFormData] = useState({
+    name: "",
+    username: "",
+    email: "",
+    password: "",
+  });
+  const validatePassword = (passwordToCheck: string) => {
+    const upper = /(.*[A-Z].*)/;
+    const lower = /(.*[a-z].*)/;
+    const number = /[0-9]/;
```

Comment:
Regex has a handy little single character test thing for this exact purpose:
```suggestion
    const number = /\d/;
```